examples/basic.org

```diff
@@ -80,7 +80,7 @@ And for numerically sequential sequences, they will be collapsed when compiled t
 [[printindex:]]
 
 [[bibliographystyle:authordate1]]
-bibliography:/Users/jkitchin/Dropbox/emacs/bibliography/references.bib
+bibliography:../org-ref.bib
```

Comment:
this might solve the nil message, but I am not sure this file has the references used in this file.